src/test/resources/test_data/clear-all-data.sql

```diff
@@ -5,4 +5,6 @@ DELETE from additional_condition_upload_summary;
 DELETE from additional_condition;
 DELETE from bespoke_condition;
 DELETE from licence_history;
-DELETE from licence;
\ No newline at end of file
+DELETE from community_offender_manager_licence_mailing_list;
+DELETE from licence;
+DELETE from community_offender_manager;
```

Comment:
Just checking this order is right. No ref. integrity preventing licences being deleted whilst community-offender-managers still exist for them? Its one-way, with licence being the parent?